Extension/src/LanguageServer/client.ts

```diff
@@ -1209,8 +1217,10 @@ export class DefaultClient implements Client {
             ],
             initializationOptions: {
                 freeMemory: os.freemem() / 1048576,
+                resetFilePathHandling: resetDatabase,
```

Comment:
'resetFilePathHandling' sounds like it's left over from the old naming ideas.  Should this be 'resetDatabase'?